modules/cfe_testcase/src/time_conversion_test.c

```diff
@@ -0,0 +1,172 @@
+/*************************************************************************
+**
+**      GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**      Copyright (c) 2006-2021 United States Government as represented by
+**      the Administrator of the National Aeronautics and Space Administration.
+**      All Rights Reserved.
+**
+**      Licensed under the Apache License, Version 2.0 (the "License");
+**      you may not use this file except in compliance with the License.
+**      You may obtain a copy of the License at
+**
+**        http://www.apache.org/licenses/LICENSE-2.0
+**
+**      Unless required by applicable law or agreed to in writing, software
+**      distributed under the License is distributed on an "AS IS" BASIS,
+**      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**      See the License for the specific language governing permissions and
+**      limitations under the License.
+**
+** File: time_conversion_test.c
+**
+** Purpose:
+**   Functional test of basic Time Conversion APIs
+**
+*************************************************************************/
+
+/*
+ * Includes
+ */
+#include "cfe_test.h"
+
+/* Check if its nearly possible that Time == Target */
+/* DUPLICATE! from time_current_test.c */
+bool TimeInRange(CFE_TIME_SysTime_t Time, CFE_TIME_SysTime_t Target, OS_time_t difference)
```

Comment:
Since this function is now used in two places, it should be moved to cfe_test.h to prevent duplicate code.

Reply:
Ok, thanks.